scipy-1.0/paper.tex

```diff
@@ -249,6 +249,35 @@ \subsection*{Numerical optimization}
 
 
 \subsection*{Statistical distributions}
+Version 1.0 of SciPy saw the addition of the energy distance algorithm in
+\texttt{scipy.stats.energy\_distance}. Energy distance is used in a variety
+of applications ranging from genetic analysis\cite{Hu2008DetectingIC} to 
+multivariate independence\cite{szekely2009}. We also added the
+first Wasserstein distance algorithm to the library in
+\texttt{scipy.stats.wasserstein\_distance}. The Wasserstein distance is often
+used in color digital image analysis \cite{192468}.
+
+We now provide the cumulative distribution function of the multivariate
+normal distribution, \\\texttt{scipy.stats.multivariate\_normal}, via the newly
+associated \texttt{logcdf} and \texttt{cdf} methods. Other recent distributions 
+added to \texttt{scipy.stats} include the Argus distribution
+in \texttt{scipy.stats.argus} (used in particle physics\cite{ALBRECHT1990278}), a histogram-based
+distribution in \texttt{scipy.stats.rv\_histogram}, the multinomial
+distribution in \texttt{scipy.stats.multinomial} (used, for example, in natural
```

Comment:
This reads like a long list from the release notes. Maybe better make `rv_histogram` stand out since it's actually a means for constructing a family of distributions? 
Ditto for `multinomial`, which is the first and so far the only multivariate discrete distribution.

Reply:
let's just delete the rest I'd say, something like `Argus` is not really big enough to mention in this paper imho